### PR TITLE
7z: update pt_BR translation

### PR DESCRIPTION
**Pull request by barbaraips**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr, Can you update the paths and some command arguments on this page to better align with the [main page](https://github.com/tldr-pages/tldr/blob/main/pages/common/7z.md) you could also refer to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) if necessary.

File: pages.pt_BR/common/7z.md [hunk 1]
```diff
@@ -19,14 +19,18 @@
...
+- Compactar utilizando um tipo específico de arquivamento/compressão:
 
-`7z i`
+`7z a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
 
 - Exibir o conteúdo de um arquivo:
 
 `7z l {{arquivo.7z}}`
+
+- Exibir os tipos de arquivamento/compressão disponíveis:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7z.md:34]: For Romance languages, the present indicative fits better
```suggestion
- Exibe os tipos de arquivamento/compressão disponíveis:
```

File: pages.pt_BR/common/7z.md [hunk 2]
```diff
@@ -19,14 +19,18 @@
...
-`7z a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7z x {{caminho/arquivo_compactado.7z}} -so`
 
-- Exibir os tipos de arquivamento/compressão disponíveis:
+- Compactar utilizando um tipo específico de arquivamento/compressão:
 
-`7z i`
+`7z a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
 
 - Exibir o conteúdo de um arquivo:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7z.md:30]:
```suggestion
- Exibe o conteúdo de um arquivo:
```

File: pages.pt_BR/common/7z.md [hunk 3]
```diff
@@ -19,14 +19,18 @@
...
 `7z x {{arquivo.7z}} -o{{caminho/diretório}}`
 
-- Compactar utilizando um tipo específico de arquivamento/compressão:
+- Descompactar um arquivo para a saída padrão:
 
-`7z a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7z x {{caminho/arquivo_compactado.7z}} -so`
 
-- Exibir os tipos de arquivamento/compressão disponíveis:
+- Compactar utilizando um tipo específico de arquivamento/compressão:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7z.md:26]:
```suggestion
- Compacta utilizando um tipo específico de arquivamento/compressão:
```

File: pages.pt_BR/common/7z.md [hunk 4]
```diff
@@ -19,14 +19,18 @@
 
 `7z x {{arquivo.7z}} -o{{caminho/diretório}}`
 
-- Compactar utilizando um tipo específico de arquivamento/compressão:
+- Descompactar um arquivo para a saída padrão:
```
Comment (navarroaxel) [inline pages.pt_BR/common/7z.md:22]:
```suggestion
- Descompacta um arquivo para a saída padrão:
```

File: pages.pt_BR/common/7z.md [hunk 5]
```diff
@@ -19,14 +19,18 @@
...
 
-`7z a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
+`7z x {{caminho/arquivo_compactado.7z}} -so`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7z.md:24]:
```suggestion
`7z x {{caminho/para/arquivo_compactado.7z}} -so`
```

File: pages.pt_BR/common/7z.md [hunk 6]
```diff
@@ -19,14 +19,18 @@
...
 
-- Exibir os tipos de arquivamento/compressão disponíveis:
+- Compactar utilizando um tipo específico de arquivamento/compressão:
 
-`7z i`
+`7z a -t{{zip|gzip|bzip2|tar}} {{arquivo_compactado.7z}} {{caminho/arquivo_ou_diretório}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7z.md:28]:
```suggestion
`7z a -t{{zip|gzip|bzip2|tar}} {{caminho/para/arquivo_compactado.7z}} {{caminho/para/arquivo_ou_diretório}}`
```

File: pages.pt_BR/common/7z.md [hunk 7]
```diff
@@ -19,14 +19,18 @@
 
 `7z x {{arquivo.7z}} -o{{caminho/diretório}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/7z.md:20]: Could you please apply this kind of changes to the other examples? Thanks
```suggestion
`7z x {{caminho/para/arquivo.7z}} -o{{caminho/para/diretório}}`
```

**Comment by barbaraips** [comment]
Pull request updated @navarroaxel @kbdharun

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**